core/src/components/input/input.md.scss

```diff
@@ -117,3 +117,27 @@
 :host(.input-shape-round) {
   --border-radius: 16px;
 }
+
+// Start/End Slots
+// ----------------------------------------------------------------
+
+/**
+ * Slotted buttons have a lot of default padding that can
+ * cause them to look misaligned from other pieces such
+ * as the control's label, especially when using a clear
+ * fill. We also make them circular to ensure that non-
+ * clear buttons and the focus/hover state on clear ones
+ * don't look too crowded.
+ */
+::slotted(ion-button[slot="start"].button-has-icon-only),
+::slotted(ion-button[slot="end"].button-has-icon-only) {
+  --border-radius: 50%;
+  --padding-start: 8px;
+  --padding-end: 8px;
```

Comment:
Does this change need to be made to `select.scss` and `textarea.scss` too?

Reply:
Oh, yes! Haha, thank you for the reminder 😆